src/Modules/SimplCommerce.Module.Localization/Areas/Localization/Controllers/LocalizationApiController.cs

```diff
@@ -47,23 +47,24 @@ public async Task<IActionResult> GetResources(string cultureId)
         {
             var resources = await _resourceRepository.Query()
                 .Where(x => x.CultureId == cultureId)
-                .Select(x => new ResourceItemVm {
+                .Select(x => new ResourceItemVm
+                {
                     Key = x.Key,
                     Value = x.Value,
                     CultureId = x.CultureId,
                     IsTranslated = true
                 })
                 .ToListAsync();
 
-            if(cultureId != STANDARD_CULTURE_ID)
+            if (cultureId != STANDARD_CULTURE_ID)
```

Comment:
Remove the extra space

Reply:
Visual Studio is only respecting the spacing preferences in the **.editorconfig**
`csharp_space_after_keywords_in_control_flow_statements = true`

Should we alter this preference?

Reply:
If is it like that just leave such comment and have a look to the others ..